kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveStores.java

```diff
@@ -0,0 +1,199 @@
+   * @param keySerde the key serde. If null, the default.key.serde config will be used
+   * @param valueSerde the value serde. If null, the default.value.serde config will be used
+   * @return a store builder that can be used to build a window store with the given options
+   *         that uses Responsive's storage for its backend
+   */
+  public static <K, V> StoreBuilder<WindowStore<K, V>> windowStoreBuilder(
+      final String name,
+      final long retentionMs,
+      final long windowSize,
+      final boolean retainDuplicates,
+      final Serde<K> keySerde,
+      final Serde<V> valueSerde
+  ) {
+    final WindowBytesStoreSupplier storeSupplier = new ResponsiveWindowedStoreSupplier(
+        name,
+        retentionMs,
+        windowSize,
+        retainDuplicates
+    );
+
+    return new ResponsiveStoreBuilder<>(
+        Stores.windowStoreBuilder(storeSupplier, keySerde, valueSerde),
+        false
+    );
+  }
+
+  /**
+   * Create a {@link StoreBuilder} that can be used to build a Responsive
+   * {@link TimestampedWindowStore} and connect it via the Processor API. If using the DSL, use
+   * {@link #windowMaterialized} instead.
+   *
+   * @param name the store name
+   * @param retentionMs the retention period in milliseconds
+   * @param windowSize the window size in milliseconds
+   * @param retainDuplicates whether to retain duplicates. Should be false for most operators
+   * @param keySerde the key serde. If null, the default.key.serde config will be used
+   * @param valueSerde the value serde. If null, the default.value.serde config will be used
+   * @return a store builder that can be used to build a window store with the given options
+   *         that uses Responsive's storage for its backend
+   */
+  public static <K, V> StoreBuilder<TimestampedWindowStore<K, V>> timestampedWindowStoreBuilder(
+      final String name,
+      final long retentionMs,
+      final long windowSize,
+      final boolean retainDuplicates,
+      final Serde<K> keySerde,
+      final Serde<V> valueSerde
+  ) {
+    final WindowBytesStoreSupplier storeSupplier = new ResponsiveWindowedStoreSupplier(
+        name,
+        retentionMs,
+        windowSize,
+        retainDuplicates
+    );
+
+    return new ResponsiveStoreBuilder<>(
+        Stores.timestampedWindowStoreBuilder(
+            storeSupplier,
+            keySerde,
+            valueSerde),
+        false
+    );
+  }
+
+  /**
+   * Create a {@link Materialized} that can be used to build a Responsive {@link KeyValueStore}
+   * and materialized in the DSL. If using the low-level Processor API, use
+   * {@link #keyValueStoreBuilder} instead.
+   *
+   * @param name the store name
+   * @return a Materialized configuration that can be used to build a key value store with the
+   *         given options that uses Responsive's storage for its backend
+   */
+  public static <K, V> Materialized<K, V, KeyValueStore<Bytes, byte[]>> materialized(
```

Comment:
what did you think about moving these into `ResponsiveMaterialized` class and naming them the same as existing methods (e.g. `as`)? That way (again) they can just `s/Materialized/ResponsiveMaterialized/g` during the migration

Reply:
😞